### PR TITLE
initial geo page and routing

### PR DESCRIPTION
**Pull request by xbexbex**
* Added blank page for the upcoming geo map
* Added routing for the page
* Added a link to the header

* Fixed linting errors in client/basePaths (default-param-last)
* Implemented eslint changes from https://github.com/openforis/fra-platform/pull/1159/commits/eed5e241b5128791716b2bdb372930ba4511a5ae to be able to commit

### CODE REVIEW

File: client/pages/Geo/geo.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@import 'client/style/partials';
```
Comment (sorja) [inline client/pages/Geo/geo.scss:1]: Add new line at EOF :)

File: client/components/LinkGeo/linkGeo.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import 'client/style/partials';
```
Comment (sorja) [inline client/components/LinkGeo/linkGeo.scss:1]: Add new line at EOF :)

File: webapp/main/basePaths.ts [hunk 3]
```diff
@@ -114,3 +117,6 @@ export const getAssessmentPrintLink = (countryIso: any, assessmentType: any, onl
 export const odp = _generate(PARAMS.countryIso, PARAMS.assessmentType, FRAGMENTS.odp, PARAMS.tab)
 export const getOdpLink = (countryIso: any, assessmentType: any, sectionName: any, odpId?: any) =>
   _generate(countryIso, assessmentType, FRAGMENTS.odp, sectionName, odpId)
+
+// ==== Geo
+export const geo = '/geo'
```
Comment (sorja) [inline webapp/main/basePaths.ts:122]: Dont change files under webapp/ unless absolutely necessary

Comment (minotogna) [reply webapp/main/basePaths.ts:122]: webapp is deprecated. or @sorja are we still using the paths from here?
in that case, for now we add the geo path here, and we'll move this later on.

@xbexbex please, make the geo a function. easier to add parameter if needed and consistent with other paths

File: client/components/LinkGeo/linkGeo.scss (hunk 2 above)
Comment (minotogna) [reply client/components/LinkGeo/linkGeo.scss:1]: @sorja @xbexbex shouldn't/didn't the pre-commit hook take care of that ?

**Comment by minotogna** [comment]
@xbexbex Let's also forget about master-geo, my mistake. I will remove this branch.
let's merge to master-meta directly

File: client/components/Header/FraHeader/FraHeader.tsx [hunk 4]
```diff
@@ -18,6 +19,7 @@ const FraHeader: React.FC = () => {
       <div className="app-header__menu">
         <LanguageSelector />
         <UserLinks />
+        <LinkGeo />
```
Comment (minotogna) [inline client/components/Header/FraHeader/FraHeader.tsx:22]: why do we have LinkGeo twice ?

File: client/components/LinkGeo/LinkGeo.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import './linkGeo.scss'
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+const LinkGeo: React.FC = () => (
+  <NavLink to="/geo" className="app-header-link-geo" exact>
```
Comment (minotogna) [inline client/components/LinkGeo/LinkGeo.tsx:6]: correct link should be /:countryIso/geo

File: client/components/LinkGeo/linkGeo.scss (hunk 2 above)
Comment (xbexbex) [reply client/components/LinkGeo/linkGeo.scss]: I guess it didn't. Fixed it manually now.

File: webapp/main/basePaths.ts (hunk 3 above)
Comment (xbexbex) [reply webapp/main/basePaths.ts]: I just removed the path from the webapp for now

File: client/components/Header/FraHeader/FraHeader.tsx (hunk 4 above)
Comment (xbexbex) [reply client/components/Header/FraHeader/FraHeader.tsx]: It was in the fraHeader for desktop layout and countrySelect in the mobile layout. I moved it to countrySelect for both.

File: client/components/LinkGeo/LinkGeo.tsx (hunk 5 above)
Comment (xbexbex) [reply client/components/LinkGeo/LinkGeo.tsx]: Okay, added this.

**Comment by xbexbex** [comment]
> @xbexbex Let's also forget about master-geo, my mistake. I will remove this branch. let's merge to master-meta directly

Okey dokey, now it's against master-meta.

**Review by minotogna — APPROVED** [review]

**Pull request merged**